### PR TITLE
Remove SyncSet in favor of Applier

### PR DESCRIPTION
**Pull request by itdove**
https://github.com/open-cluster-management/backlog/issues/6468
https://github.com/open-cluster-management/backlog/issues/10576
Signed-off-by: Dominique Vernier <dvernier@redhat.com>

### CODE REVIEW

File: pkg/controller/managedcluster/managedcluster_import_detach.go [hunk 1]
```diff
@@ -23,35 +23,69 @@ import (
...
 
 func (r *ReconcileManagedCluster) importCluster(
 	managedCluster *clusterv1.ManagedCluster,
+	clusterDeployment *hivev1.ClusterDeployment,
 	autoImportSecret *corev1.Secret) (res reconcile.Result, err error) {
 	res = reconcile.Result{}
-	var client client.Client
+
+	//Assuming that is a local import
+	client := r.client
```
Comment (hanqiuzh) [inline pkg/controller/managedcluster/managedcluster_import_detach.go:37]: does that mean we will always import local? even for manual import scenario?

Comment (itdove) [reply pkg/controller/managedcluster/managedcluster_import_detach.go:37]: The importCluster is called only if clusterDeployment or autoImportSecret present or managedCluster has the label for local-import.
https://github.com/itdove/managedcluster-import-controller/blob/05910be5a8ff93d4c3[…]216d/pkg/controller/managedcluster/managedcluster_controller.go
Line 292

**Pull request merged**